src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.ts

```diff
@@ -72,6 +100,10 @@ export class FlowsTableComponent implements OnInit, OnChanges {
     }
 
     ngOnInit(): void {
+        if (this.accountView) {
+            this.DISPLAY_COLUMNS.splice(3, 0, "dataset");
+            this.INITIATORS = [FilterByInitiatorEnum.All, FilterByInitiatorEnum.System];
+        }
```

Comment:
Looks like these should be ports too, as behavior is different.

Reply:
Removed, created universal port object